core/templates/dev/head/services/StateTopAnswersStatsService.js

```diff
@@ -50,24 +52,53 @@ oppia.factory('StateTopAnswersStatsService', [
       });
     };
 
+    $rootScope.$on('addState', function(event, args) {
+      if (stateTopAnswerStatsCache.hasOwnProperty(args.state_name)) {
```

Comment:
Why would this case happen? Would expect an exception to be thrown here or an error to be logged, it seems invalid.

Reply:
Done.